app/components/log-drain-actions/template.hbs

```diff
@@ -3,6 +3,6 @@
     <button class="btn btn-default btn-xs" {{action "restart" logDrain}}>Restart</button>
   {{/if}}
   {{#unless logDrain.hasBeenDeprovisioned}}
-    <button class="btn btn-default btn-xs" {{action "deprovision" logDrain}}>Deprovision</button>
+    <button class="btn btn-default btn-xs" {{action "deprovision" logDrain}}>Delete</button>
```

Comment:
This does a better job of setting expectations. It gets deprovisioned and then deleted via the operation.